docker/files/run/run.sh

```diff
@@ -144,11 +161,13 @@ function start_new_public_network() {
   alphanet) ID="<network>.algodev.network" ;;
   devnet) ID="<network>.algodev.network" ;;
   *)
-    echo "Unknown network"
-    exit 1
```

Comment:
When using a PEER_ADDRESS, you don't need a NETWORK or bootstrap ID. I'll change some of the conditions so that this is clear, and add this exit back in when the NETWORK option is used.